src/core/include/openvino/op/constant.hpp

```diff
@@ -569,11 +585,19 @@ class OPENVINO_API Constant : public Op {
         output.resize(element_number);
     }
 
+    template <element::Type_t Type,
+              typename StorageDataType = fundamental_type_for<Type>,
+              typename std::enable_if<Type != element::Type_t::string, bool>::type = true>
+    void fill_data(const std::string& value) {
+        OPENVINO_THROW("Called fill_data(std::string) with non-string element_type");
```

Comment:
can we print actual type of constant? can be useful for debug purposes

Reply:
Fixed.